src/toil/wdl/wdl_functions.py

```diff
@@ -20,20 +20,19 @@
 import subprocess
 import textwrap
 import uuid
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Dict, List, Any, Optional, Tuple, Union
 
 from toil.fileStores.abstractFileStore import AbstractFileStore
-from toil.wdl.wdl_types import WDLFile, WDLPair
+from toil.wdl.wdl_types import WDLType, WDLFile, WDLPair
 from toil.lib.conversions import bytes_in_unit
-from toil.lib.resources import glob  # type: ignore
```

Comment:
It looks like this function was imported to be re-exported and dropping this broke the tests. Maybe the import needs a comment to that effect?

Reply:
If the WDL tests need `glob`, why don't they import it directly?